libconsensus/rotating_pbft/RotatingPBFTEngine.h

```diff
@@ -22,6 +22,7 @@
  */
 #pragma once
 #include <libconsensus/pbft/PBFTEngine.h>
+#include <libsync/TreeTopology.h>

```

Comment:
DO NOT depend on libsync, plz move TreeTopology.h to lower layer

Reply:
Maybe we can put it in `libdevcore`.

Reply:
> Maybe we can put it in `libdevcore`.

Yes, you're right, I will move `TreeTopology.h` to `devcore`